Zoo/Workers/Worker.cs

```diff
@@ -0,0 +1,79 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Zoo.Animals;
+
+namespace Zoo.Workers
+{
+    public abstract class Worker : Person
+    {
+        Dictionary<int, Area> WorkAreaTime;
+        public Area WorkArea;
+        public int WorkTime; //seconds
+
+        protected Worker(string name) : base(name)
+        {
+        }
+
+        //In order to keep OOP, workerer types that delay a tour will not allow tourists,
+        //to continue their tour while working (prevents asking in zoo class which worker type is working)
```

Comment:
```suggestion
        /*In order to keep OOP, worker types that delay a tour will not allow tourists,
        to continue their tour while working (prevents asking in zoo class which worker type is working)*/
```

Besides multiline comment correction, always remember some people are dumb and don't understand typos. Before pushing to repo in general and to master specifically, go over all your code- including comments.